lib/errors.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = (translate, options) => {
+
+  var logger = (options || {}).logger;
+
+  if (typeof translate !== 'function') {
+    throw new Error('Errorhandler needs a translate function');
+  }
+
+  return (err, req, res) => {
+    var content = {};
+
+    if (err.code === 'SESSION_TIMEOUT') {
+      content.title = translate('errors.session.title');
+      content.message = translate('errors.session.message');
+    }
+
+    if (err.code === 'NO_COOKIES') {
+      err.status = 403;
+      content.title = translate('errors.cookies-required.title');
+      content.message = translate('errors.cookies-required.message');
+    }
+
+    content.title = content.title || translate('errors.default.title');
+    content.message = content.message || translate('errors.default.message');
```

Comment:
Security? I suppose it might expose something about our system. I don't know, I am happy to leave it as is, @easternbloc @joefitter ?